jobs/setup-for-incremental-updates.yml

```diff
@@ -0,0 +1,45 @@
+# Template that takes in the following parameters
+# ${siteName}
+# ${partitionNum}
+# The purpose of this job is to refresh all of the DW table that are shared across multiple servers
+type: "job-pipeline"
+description: "Setting up to support incremental updates for ${siteName} using partition ${partitionNum}"
+configuration:
+  jobs:
+    - type: "create-table"
+      description: "Creating ${tableName} to track incremental updates"
+      parameters:
+        tableName: "petl_incremental_update_log"
+      configuration:
+        source:
+          sqlFile: "sql/schemas/${tableName}.sql"
+        target:
+          datasource: "warehouse.yml"
+          tableName: "${tableName}"
+
+    - type: "sqlserver-bulk-import"
```

Comment:
This loads in the db_event table from the source into patient_last_update_date in the warehouse.  This is what is used to determine what patients have changed since the last watermark in any incremental job.